testproject/default.nix

```diff
@@ -0,0 +1,25 @@
+{ pkgs ? import <nixpkgs> {} }:
```

Comment:
these testproject files need to be removed.

Reply:
Doh! I thought I did but probably got confused by a rebase.